planetarium/NineChronicles
# Prepare arena season 0

### Description

1. fix daily arena interval 500 -> 1600
2. add season medal item
3. set the season medal randomly drop in arena battle (ratio=30%)

### How to test

1. check the file changes
2. check the screenshot

### Related Links

[아레나 매 전투마다 랜덤하게 시즌메달 아이템 드랍되게](https://app.asana.com/0/958521740385861/1200833547455893/f)
[티켓 발행주기 및 아레나 갱신주기 조절](https://app.asana.com/0/958521740385861/1200799644541028/f)

### Required Reviewers

@planetarium/9c-dev @ipdae @longfin @kijun @jckdotim @plafina2 

### Screenshot

<img width="526" alt="스크린샷 2021-08-31 오후 10 48 53" src="https://user-images.githubusercontent.com/41294407/131515342-5637362d-cfc8-4712-a2c3-42795005e9aa.png">
<img width="215" alt="스크린샷 2021-08-31 오후 10 49 08" src="https://user-images.githubusercontent.com/41294407/131515367-798c22df-c382-4c27-ad23-d2fe692d4ecf.png">
